Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/LocalViewBookmarkLoader.cpp

```diff
@@ -45,97 +177,174 @@ namespace AzToolsFramework
         return outputFile.Length() > 0;
     }
 
-    static AZStd::string BookmarkSetRegPath(const AZStd::string& bookmarkFileName)
+    static AZStd::string LocalViewBookmarkSetRegPath(const AZStd::string& bookmarkFileName)
     {
         return AZStd::string::format("/%s/LocalBookmarks", bookmarkFileName.c_str());
     }
 
-    static AZStd::string BookmarkSetRegPathAtIndex(const AZStd::string& bookmarkFileName, const int index)
+    static AZStd::string LocalViewBookmarkSetRegPathAtIndex(const AZStd::string& bookmarkFileName, const int index)
     {
-        return AZStd::string::format("%s/%i", BookmarkSetRegPath(bookmarkFileName).c_str(), index);
+        return AZStd::string::format("%s/%i", LocalViewBookmarkSetRegPath(bookmarkFileName).c_str(), index);
     }
 
-    void LocalViewBookmarkLoader::RegisterViewBookmarkLoaderInterface()
+    static AZStd::string LastKnownLocationSetRegPath(const AZStd::string& bookmarkFileName)
     {
-        AZ::Interface<ViewBookmarkLoaderInterface>::Register(this);
+        return "/" + bookmarkFileName + "/LastKnownLocation";
```

Comment:
nit: This would do only 1 string operation instead of 2:

````
return AZStd::string::format("/%s/LastKnownLocation", bookmarkFileName.c_str());
````

Take it or leave it ;)

Reply:
Good call! This got refactored from somewhere else, this is much nicer, thanks!